cloud/pkg/cloudhub/config/config.go

```diff
@@ -56,8 +56,6 @@ func InitConfigure(hub *v1alpha1.CloudHub, kubeAPIConfig *v1alpha1.KubeAPIConfig
 		if ca != nil && caKey != nil {
 			Config.Ca = ca
 			Config.CaKey = caKey
-		} else if !(ca == nil && caKey == nil) {
```

Comment:
Why delete this line?

Reply:
> Why delete this line?

sorry, i will revert it. i ingore the situation that it does't have complete certificates